tests/integration/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+  girder:
+    image: girder/girder:gw_integration_test
```

Comment:
Can build be used here as well? So `docker-compose build` can be used to build all the relevant Dockerfiles instead of pulling?